src/test/kotlin/uk/gov/dluhc/printapi/testsupport/assertj/assertions/AnonymousElectorDocumentCertificateAssert.kt

```diff
@@ -39,4 +60,35 @@ class AnonymousElectorDocumentCertificateAssert
 
         return this
     }
+
+    fun hasDelivery(delivery: Delivery): AnonymousElectorDocumentCertificateAssert {
+        isNotNull
+
+        DeliveryAssert(actual!!.delivery)
```

Comment:
minor (non-blocking) - I'm slightly surprised you didn't use your usual pattern of `with(delivery) {...}` :)

Reply:
I have `renamed` the function param and I think earlier I did a copy paste of this function.
However without with, it's looking as more readable I think :)